File: Java/백준/Bronze/26068. 치킨댄스를 추는 곰곰이를 본 임스 2/치킨댄스를 추는 곰곰이를 본 임스 2.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int q = in.nextInt(); int sum=0;
        for(int i =0; i<q; i++){
            int w = Integer.parseInt(in.next().substring(2));
            if(w<=90){
                sum++;
            }
        }
        System.out.println(sum);
    }
}

File: C#/백준/Bronze/26068. 치킨댄스를 추는 곰곰이를 본 임스 2/치킨댄스를 추는 곰곰이를 본 임스 2.cs
using System;

namespace q{
    class w{
        static void Main(){
            int q = int.Parse(Console.ReadLine()); int sum=0;
            for(int i =0; i<q; i++){
                int a = int.Parse(Console.ReadLine().Substring(2));
                if(a<=90){
                    sum++;
                }
            }
            Console.WriteLine(sum);
        }
    }
}